packages/reason-css-parser/ppx/Reason_css_parser_ppx.re

```diff
@@ -59,4 +59,26 @@ let valueRecExtension =
     expander(~recursive=true),
   );
 
-Driver.register_transformation(~extensions=[valueExtension, valueRecExtension], "css-value-parser-ppx");
+let gen_type = (str) => {
+  let bindings = List.find_opt(fun
+    | {pstr_desc: Pstr_value(Recursive, _), pstr_loc: loc} => {
+      let filename = loc.loc_start.Lexing.pos_fname;
+      filename == "packages/reason-css-parser/test/spec.re";
```

Comment:
Ppx doesn't run on Tests because of this

Reply:
I removed it, but do we want to run this in all files that contains `the  ppx? Or only in specific files? If its the second option, what do you suggest to achieve this? I think the dune `per_module` + an specific flag (`--gen-types` ?) would be nice, WDYT?

Reply:
For now reason-css-parser's ppx are meant to be running on the same single file. I wouldn't worry too much, since there aren't any other invocations of this ppx elsewhere.